models/media.go

```diff
@@ -503,6 +550,13 @@ func (m *Mediafile) ObtainVideoCodec() []string {
   return nil
 }
 
+func (m *Mediafile) ObtainVframes() []string {
+  if m.vframes != 0 {
+    return []string{"-vframes",fmt.Sprintf("%d", m.vframes)}
```

Comment:
-vframes number (output)
Set the number of video frames to output. **This is an obsolete alias for -frames:v, which you should use instead.**